### ISSUE TITLE
Report hasn't the TOTAL line if one file is tested only 

### ISSUE THREAD

**Issue by mosling** [coveragepy/coveragepy#922]
**Describe the bug**
A very simple setup, I have one python script (script.py) and a test for this script (script_test.py).

The result for `coverage run -m pytest` and `coverage report` is

```
Name            Stmts   Miss  Cover
-----------------------------------
adder.py            2      0   100%
adder_test.py       7      1    86%
-----------------------------------
TOTAL               9      1    89%
```
But the same with `coverage run --omit='*_test.py' -m pytest`  and `coverage report` results into

```
Name       Stmts   Miss  Cover
------------------------------
adder.py       2      0   100%
```

The summary line with TOTAL is missing
I use GitLab and look for the TOTAL line to catch the test coverage.

**To Reproduce**
How can we reproduce the problem?  Please *be specific*.
1. Python 3.8.0
1. Coverage.py, version 5.0.3 with C extension
1. Example project at https://github.com/mosling/coverage-example

**Expected behavior**
TOTAL line should appear in every case. It is important for data processing to have the same behavior in each case, I never see the output ;-)

**Comment by nedbat** [comment]
This has been reported before, in #732.  My advice there was to simply read the last word on the last line of the report, which will be what you want in all cases.  But now, as of version 5.0, there's a better answer: use the "coverage json" report, and read the number programmatically.

**Comment by nedbat** [comment]
@mosling Also, I'm curious what you use the total number for.  Maybe there's a better way we can provide it.  Do you know about the ``--fail-under`` option?

**Comment by nedbat** [comment]
Now with #1091, the TOTAL line is always output.

**Comment by nedbat** [comment]
This is now released as part of [coverage 5.4](https://pypi.org/project/coverage/5.4).
